tracy/cs/guide.texy

```diff
@@ -146,12 +146,18 @@ Pokud používát Nette Framework, podívejte se, jak [nastavit režim pro něj
 Logování chyb
 =============
 
-V produkčním režimu Laděnka automaticky všechny chyby a zachycené výjimky zaznamenává do textového logu. Cestu k němu nastavíme tímto způsobem:
+V produkčním režimu Laděnka automaticky všechny chyby a zachycené výjimky zaznamenává do souboru error.log. Aby logování mohlo probíhat, musíme nastavit cestu k logovacímu adresáři:
 
 ```php
 Debugger::$logDirectory = __DIR__ . '/log';
 ```
 
+nebo jako druhý parametr metody enable():
+
+```php
+Debugger::enable( ..., logDirectory: $rootDir.'/log');
```

Comment:
Když používáš named argument, tak stačí ten. `...` je v php pro variadika a first class callables a tohle není ani jeden z těch případů. Zkus dodržovat i formátování.

```suggestion
Debugger::enable(logDirectory: $rootDir . '/log');
```

Případně by to šlo takto, netuším zda už Tracy vyžaduje PHP 8 (kód v dokumentaci by měl odpovídat minimální podporované verzi)

```suggestion
Debugger::enable(null, $rootDir . '/log');
```

Reply:
Právě protože je ten zápis komplikovaný, zůstal bych jen u té proměnné logDirectory.

Reply:
"Když používáš named argument" - napsal jsem to blbě, ale ten druhý způsob tam být nemusí.